plugins/content_distribution/admin/forms/ConfigurableProfileConfiguration.php

```diff
@@ -5,6 +5,23 @@
  */
 abstract class Form_ConfigurableProfileConfiguration extends Form_ProviderProfileConfiguration
 {
+	public function init ()
+	{
+		parent::init();
+		
+		$this->addElement('checkbox', 'exclude_legacy_categories', array(
+			'label' => 'Exclude legacy categories',
```

Comment:
I'm sure you could find a text that actually explains the feature, e.g. "Enable categories with privacy-context".